src/api/requests/AddressCreateRequest.ts

```diff
@@ -0,0 +1,20 @@
+import { IsNotEmpty } from 'class-validator';
+import { RequestBody } from '../../core/api/RequestBody';
+
+
+export class AddressCreateRequest extends RequestBody {
+    // @IsNotEmpty()
+    // public title: string;
```

Comment:
uncomment these, and I think at least address_line_1, city and country should be annotated with IsNotEmpty